charts/umbrella/values-adopter-data-exchange-stub.yaml

```
@@ -0,0 +1,212 @@
+      ingressClassName: "nginx"
+      hostname: "centralidp.tx.test"
+      annotations:
+        nginx.ingress.kubernetes.io/cors-allow-credentials: "true"
+        nginx.ingress.kubernetes.io/cors-allow-methods: "PUT, GET, POST, OPTIONS"
+        nginx.ingress.kubernetes.io/cors-allow-origin: "http://centralidp.tx.test"
+        nginx.ingress.kubernetes.io/enable-cors: "true"
+        nginx.ingress.kubernetes.io/proxy-buffer-size: "128k"
+        nginx.ingress.kubernetes.io/proxy-buffering: "on"
+        nginx.ingress.kubernetes.io/proxy-buffers-number: "20"
+        nginx.ingress.kubernetes.io/use-regex: "true"
+      tls: false
+    # -- uncomment the following for persistence
+    # postgresql:
+    #   primary:
+    #     persistence:
+    #       enabled: true
+
+
+bdrs-server-memory:
+  enabled: true
+  seeding:
+    bpnList:
+      - bpn: "BPNL000000000000"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL000000000000"
+      - bpn: "BPNL00000003B3NX"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B3NX"
+      - bpn: "BPNL00000003CSGV"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003CSGV"
+      - bpn: "BPNL00000003B6LU"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B6LU"
+      - bpn: "BPNL00000003AXS3"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AXS3"
+      - bpn: "BPNL00000003AZQP"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AZQP"
+      - bpn: "BPNL00000003AWSS"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AWSS"
+      - bpn: "BPNL00000003AYRE"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AYRE"
+      - bpn: "BPNL00000003AVTH"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AVTH"
+      - bpn: "BPNL00000000BJTL"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000000BJTL"
+      - bpn: "BPNL00000003CML1"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003CML1"
+      - bpn: "BPNL00000003B2OM"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B2OM"
+      - bpn: "BPNL00000003B0Q0"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B0Q0"
+      - bpn: "BPNL00000003B5MJ"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B5MJ"
+      - bpn: "BPNS0000000008ZZ"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNS0000000008ZZ"
+      - bpn: "BPNL00000003CNKC"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003CNKC"
+      - bpn: "BPNS00000008BDFH"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNS00000008BDFH"
+  hostname: &hostname bdrs-server.tx.test
+  server:
+    trustedIssuers:
+      - did:web:ssi-dim-wallet-stub.tx.test:BPNL000000000000
+    ingresses:
+      - enabled: true
+        hostname: *hostname
+        endpoints:
+          - directory
+          - management
+        tls:
+          enabled: false
+
+dataconsumerOne:
+  enabled: true
+  secrets:
+    edc-miw-keycloak-secret: BPNL00000003AZQP
```

it's a bit strange to usage a BPN as client secret, isn't it?
I assume due to the stub it's irrelevant what we put there
```suggestion
    edc-miw-keycloak-secret: changeme
```
Just fyi, I created an issue to follow up on the naming of the parameter https://github.com/eclipse-tractusx/tractus-x-umbrella/issues/130

---

Should I change the names in this PR, or will you do it in another PR?"

---

I just noticed this while reviewing, it should be part of a separate PR. For now I just created an issue to raise awareness, if you'd like to pick that up at some later point, that would be great.